server/src/main/java/io/druid/server/security/AuthenticationUtils.java

```diff
@@ -33,7 +33,9 @@ public static void addAuthenticationFilterChain(
   )
   {
     for (Authenticator authenticator : authenticators) {
-      FilterHolder holder = new FilterHolder(authenticator.getFilter());
+      FilterHolder holder = new FilterHolder(
```

Comment:
The AuthenticationWrappingFilter change should involve a small doc change to how the authentication chain works (first one to identify the user is respected, and then others are skipped).